nautobot/users/views.py

```diff
@@ -250,30 +248,45 @@ class SavedViewUIViewSet(
     table_class = SavedViewTable
     action_buttons = ("export",)
 
-    def get_required_permission(self):
+    def get_queryset(self):
         """
-        Obtain the permissions needed to perform certain actions on a model.
+        Get the list of items for this view.
+        All users should be able to see saved views so we do not apply extra permissions.
```

Comment:
This too would need to restrict to views owned by the user, right?

Reply:
I am not sure this is needed since `<app_label>.view_<model_name>` permissions are enforced at each list view. Also it is difficult to handle the case of `AnonymousUser`. Tried `self.queryset.filter(owner=None)` and it does not work.